pcap/pcap_windows.go

```diff
@@ -7,18 +7,712 @@
 
 package pcap
 
-/*
-#include <pcap.h>
-*/
-import "C"
-
 import (
 	"errors"
+	"fmt"
 	"os"
 	"runtime"
+	"sync"
+	"syscall"
+	"time"
 	"unsafe"
+
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+)
+
+const npcapPath = "\\Npcap"
+
+func initDllPath() {
```

Comment:
can this function store the path it ends up using and supply that in the errors returned by mustLoad/mightLoad to be more specific about exactly which (full) paths it's having trouble loading things from?

Reply:
Not directly, but possible via kernel32.GetModuleFileName; Added this code (with fallback to old behaviour if this doesn't work)